Jenkinsfile

```diff
@@ -43,7 +43,7 @@ pipeline {
         stage("Unit test") {
           steps {
             echo "Starting unit test execution."
-            sh "./scripts/execute_unittest.sh ${env.testWithDatabase}"
+            // sh "./scripts/execute_unittest.sh ${env.testWithDatabase}"
```

Comment:
Revert these commented changes. I know you are doing it, just keeping for record so that it won't get missed.

Reply:
done